htdocs/comm/propal/card.php

```diff
@@ -2967,7 +2967,7 @@
 		print $formfile->showdocuments('propal', $objref, $filedir, $urlsource, $genallowed, $delallowed, $object->model_pdf, 1, 0, 0, 28, 0, '', 0, '', $soc->default_lang, '', $object);
 
 		// Show links to link elements
-		$linktoelem = $form->showLinkToObjectBlock($object, null, array('propal'));
```

Comment:
Instead of disabling the param saying which kind of element it is at each call, i suggest instead to keep caller unchanged and add into the method showLinkToObjectBlock a test on the hidden option MAIN_ALLOW_LINK_BETWEEN_ALL_OBJ_TYPE (that will allow link to same type of element), if on, you reset the array $excludelinksto.
Linking between same type may bring troubles, so it is better to start slowly by allowing it with a hidden option.